src/components/modals/ActionScorer.tsx

```diff
@@ -40,6 +41,38 @@ interface IRenderableColumn extends OF.IColumn {
     render: (actionForRender: ActionForRender, component: ActionScorer, index: number) => React.ReactNode
 }
 
+const findNumberFromMemory = (memory: CLM.Memory, isMultivalue: boolean): number | undefined => {
+    if (isMultivalue) {
+        return memory.entityValues.length
+    }
+
+    const valueString: string | undefined = memory
+        && memory.entityValues[0]
+        && (memory.entityValues[0].resolution ? true : undefined)
+        && (memory.entityValues[0].resolution as any).value as string
+
+    const value = valueString
+        ? parseInt(valueString)
+        : undefined
```

Comment:
Should this throw an error on non-numeric values ?

Reply:
Hmm, I didn't think so. Think it's better to be flexible here and take in any type of memory value otherwise we just add more layers.  Wonder what others think.